projects/cashmere/src/lib/sass/tables.scss

```
@@ -26,10 +26,10 @@ $row-selected-font-color: $white;
 $row-selected-border-color: $white;
 
 table.hc-table {
-    border: $table-border;
     width: 100%;
     max-width: 100%;
     color: $tbody-font-color;
+    border: 1px solid transparent;
```

We could make a variable for this border combo:
```suggestion
    border: $table-border-transparent;
```